start-from-zero/index.css

```diff
@@ -246,21 +350,208 @@ textarea {
 
 #select {
     float: right;
+    -webkit-appearance: none; 
+    -moz-appearance: none;
+    appearance: none;   
+    background-image: url(assets/down-arrow.png);
+    background-repeat: no-repeat;
+    background-position: 90% 50%;
+    background-size: 13px;
+    position: relative;
+    height: 20px;
+    
+}
+
+
+
+textarea {
+    width: 99.7%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+
+#footer-new {
+    position: relative;
 }
 
 fieldset {
+    margin: 30px 0;
+}
+
+.fieldset-wrap {
+    padding: 10px 10px;
+}
+
+fieldset legend {
+    padding: 0 20px;
+}
+
+#inner-li {
+    height: 100%;
+}
+
+.ultimate-wrap {
     width: 100%;
+    margin: 0 auto;
 }
 
-#footer-new {
+.outer-ul, .down-ul {
```

Comment:
While this is OK(-ish) for the purposes of a training project, it's seldom a good idea to couple class names so tightly with the markup they're targeting. That is because both markup and styling have a tendency to evolve over time. So, what is a `<ul>` today, might become an `<ol>` or a `<div>` or a `<section>` etc a year from now.

To keep things truly decoupled, we have to learn to think in more generic terms. What are these classes going to be used for? What is their intended behavior? Let's see if we can come up with a more abstract name. (Btw, this has got to be one of the aspects that most developers struggle with, at least initially.)